src/qaqc/download_and_flag_fxns/add_seasonal_flag.R

```diff
@@ -14,19 +14,15 @@ add_seasonal_flag <- function(df) {
 
   df <- df %>%
     # Using seasonal cut-offs...
-    left_join(lookup, by = "season") %>%
-    select(!c(site.y, parameter.y),
-           site = site.x,
-           parameter = parameter.x) %>%
-    # ... flag obs that are outside the seasonal 1-99 percentile range:
+    left_join(lookup, by = c("site","parameter","season")) %>%
+    # ... flag obs that are outside the seasonal 1st - 99th percentile range:
     add_flag((mean < t_mean01 | mean > t_mean99), "outside of seasonal range") %>%
-    # flag obs whose slope is greater than the 99th percentile range
-    add_flag((slope_ahead >= t_slope_behind_99 | slope_behind >= t_slope_behind_99), "slope violation") %>%
+    # flag obs whose slope is outside the 1st - 99th percentile range
+    add_flag((slope_ahead >= t_slope_behind_99 | slope_behind >= t_slope_behind_99 |
+              slope_ahead <= t_slope_behind_01 | slope_behind <= t_slope_behind_01), "slope violation") %>%
```

Comment:
question: I thought that slope_behind_01 would be the tamest/smallest slopes? Were you able to generate the flag report after you did this to see if the percentage of slope violations increased?

Reply:
Because we are no longer getting the abs(slope) the 1st percentile slope will actually be a negative number now, and therefore can be considered an extreme case of a negative slope...I think? Could you check that out for me?